style.css

```diff
@@ -0,0 +1,73 @@
+body{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.container{
```

Comment:
Add space after the class name and just before the opening curly brace:

this `.container {` instead of this `.container{`

Reply:
Done